pkg/cli/env.go

```diff
@@ -23,18 +23,18 @@ var (
 	cli_gitdirty = ""
 )
 
-type CliEnv struct {
+type CompiledEnv struct {
 	Name     string
```

Comment:
Why not pick `os.Env[0]` everywhere this is needed? Or at least as the set value (rather than complicate with a build flag)

Reply:
voiced on this, the ldflag is good enough for now. We can revisit in the future